### PR TITLE
Add chain 103090

### PR DESCRIPTION
**Pull request by billaure-crystaleum**
Hi yah for Crystaleum 💎 the changes requested are done now ready for merge

### CODE REVIEW

**Comment by billaure-crystaleum** [comment]
@kuzdogan this PR is ready for merge my friend

File: services/core/src/sourcify-chains.ts [hunk 1]
```diff
@@ -424,5 +424,10 @@ export default {
         "monitored": false,
         "contractFetchAddress": `https://subnet-explorer-api.avax-test.network/v1.1/432201/` + AVALANCHE_SUBNET_SUFFIX,
     },
+    "103090": { // Crystaleum Mainnet
+        "supported": true,
+        "monitored": false,
+        "contractFetchAddress": "https://scan.crystaleum.org/" + BLOCKSCOUT_SUFFIX,
+    },
```
Comment (kuzdogan) [inline services/core/src/sourcify-chains.ts:432]: Can you add `"txRegex": getBlockscoutRegex()` to the object for immutable verification?

Comment (billaure-crystaleum) [reply services/core/src/sourcify-chains.ts:432]: ok

**Comment by billaure-crystaleum** [comment]
Hi yah, done thanks.

**Comment by kuzdogan** [comment]
All good :+1: merged

**Comment by kuzdogan** [comment]
Turning off support as the chain seems to be shut down

**Pull request merged**